### PR TITLE
CICD2 Migration... and more

### PR DESCRIPTION
**Pull request by briansaildrone**
This PR:
* migrates titiler to CICD2
* moves titiler to codeartifact
* moves titiler to a saildrone managed base image
* tests out some fancy new orb stuff such as tag suffixes
* omits the `/app` directory for the codebase, since this is a forked repo

a follow up PR will adjust the orb version, as the changes need to be tested against the full feature > main (sd-master in this case) deploy pipeline

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,12 @@
-ARG BASE_IMAGE=python:3.8.9-slim-buster
+ARG BASE_IMAGE=026314688606.dkr.ecr.us-west-2.amazonaws.com/python:3.8
```
Comment (tcjennings) [inline Dockerfile:1]: Please set the # syntax directive for buildkit frontend v1

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[tool.poetry]
+name = "titiler"
+version = "0.1.0"
+description = "Automated background task service."
+authors = ["Jeff Wang <jeff@saildrone.com>"]
```
Comment (tcjennings) [inline pyproject.toml:5]: Please set the authors field to a group instead of an individual.

**Review by tcjennings — CHANGES_REQUESTED** [review]
Please rebase these commits to a single commit (e.g., using `git rebase -i`) and consider changing the branch name to `sd-main`.

**Comment by briansaildrone** [comment]
@Toby Jennings ***@***.***>  can you handle the suggested
changes?
* Unsure what group y'all use
* I'm indifferent to branch and this is a data pipeline owned project
* you're the buildkit pro

Thanks!

*Brian Derdiarian*
Engineering Manager, Platform Infrastructure
Saildrone, Inc.
<https://streaklinks.com/BV1j40ENjZX2fpyXOw46M-K8/https%3A%2F%2Fwww.saildrone.com%2F>
1050 W. Tower Avenue Alameda, CA 94501

Follow us on social @saildrone!
FACEBOOK
<https://streaklinks.com/BV1j40IgIJdpmDN_JQ0mx0PT/https%3A%2F%2Fwww.facebook.com%2Fsaildrone>
 | TWITTER
<https://streaklinks.com/BV1j40I1tUfDF-wkqAXayBkq/https%3A%2F%2Ftwitter.com%2F%40saildrone>
 | LINKEDIN
<https://streaklinks.com/BV1j40EICIiqsCfyJwU1_30o/https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Fsaildrone-inc%2F>
 | INSTAGRAM
<https://streaklinks.com/BV1j40I4OMPmD89N1gumojmz/https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Fsaildrone-inc%2F>
 | YOUTUBE
<https://streaklinks.com/BV1j40EWFpBE3OybHAgXoVLA/https%3A%2F%2Fwww.youtube.com%2Fsaildrone>

**Review by tcjennings — APPROVED** [review]
That oughta do it for now

**Review by alicehuang-saildrone — APPROVED** [review]
Lgtm!

**Pull request merged**